example/plugin/toerror/toerror.go

```diff
@@ -0,0 +1,57 @@
+package toerror
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+)
+
+func upgradeUser(endpoint, username string) error {
```

Comment:
Maybe we can come up with a better example, where we really need to a lookup in a map and then do something that might cause an error.

Reply:
replaced with [Lookup](https://golang.org/pkg/reflect/#StructTag.Lookup) method from reflect package
resolving it